src/components/course/CoursesDataTable.vue

```diff
@@ -118,7 +121,10 @@
         <tbody v-if="!items.length">
           <tr>
             <td id="message-when-zero-courses" class="ma-4 text-no-wrap title" :colspan="headers.length">
-              {{ messageWhenZeroCourses }}
+              <span v-if="refreshing">
+
+              </span>
```

Comment:
@johncrossman Does this empty `span` do anything?

Reply:
Whoops. Not a thing. Amending...